cryptoki/src/types/object.rs

```diff
@@ -506,6 +518,7 @@ impl TryFrom<CK_KEY_TYPE> for KeyType {
     fn try_from(key_type: CK_KEY_TYPE) -> Result<Self> {
         match key_type {
             CKK_RSA => Ok(KeyType::RSA),
+            CKK_EC => Ok(KeyType::EC),
```

Comment:
Could you also map `CKK_ECDSA` to `KeyType::EC`? The spec seems to treat them similarly, e.g. `EC (also related to ECDSA) private key objects (object class CKO_PRIVATE_KEY, key type CKK_EC or CKK_ECDSA) hold EC private keys`

Reply:
Unfortunately I can't do that because... `CKK_ECDSA` is exactly the same value as `CKK_EC`: https://github.com/parallaxsecond/rust-cryptoki/blob/main/cryptoki-sys/src/lib.rs#L133 and fighting the compiler is a futile battle!

Reply:
Ah, that makes sense - well, at least in terms of our interface, thanks!